libdnf.spec

```diff
@@ -154,6 +155,7 @@ popd
 %{_libdir}/pkgconfig/%{name}.pc
 %{_includedir}/%{name}/
 %{_datadir}/gir-1.0/Dnf-*.gir
+%{_datadir}/vala/vapi/
```

Comment:
@kalev Do we? I've seen it going on in Fedora packages. For example, appstream has `appstream-vala`...